pom.xml

```
@@ -51,7 +51,7 @@
     <maven.deploy.plugin.version>2.8.2</maven.deploy.plugin.version>
 
     <!--Backend properties-->
-    <ddf.version>2.23.0</ddf.version>
+    <ddf.version>2.24.0-SNAPSHOT</ddf.version>
```

```suggestion
    <ddf.version>2.24.0</ddf.version>
```
This suggestion will make sure CI wont pass until 2.24.0 released, which is probably preferable now that we've tested that it works with the snapshot version.